opendse-model/src/main/java/net/sf/opendse/model/Models.java

```diff
@@ -41,8 +41,8 @@
 public class Models {

```

Comment:
Please check for formatting style. It seems all the comments are touched by your style.